RELEASE-PROCESS.MD

```diff
@@ -33,14 +36,19 @@ KEDA Deployment YAML file (eg. keda-2.4.0.yaml) is also automatically created an
 
 ### Release template
 
-Every release should use the provided template below to create the GitHub release and ensure that a new GitHub Discussion is created.
+Every release should use the template provided below to create the GitHub release and ensure that a new GitHub Discussion is created.
 
-> 💡 Don't forget to update the version & new contributors in the template
+> ### 💡 IMPORTANT
+> 
+> Remember to make the following changes to the template:
+> 
+> - Replace `INSERT-CORRECT-VERSION` (there are **two** occurrences in the template) with the new-release ID
+> - Update the list of new contributors
```

Comment:
Can you give some info here or is it in the README for docs?

Reply:
I don't know. This was part of your original release-process text, I just reformatted it. Can I leave this up to y'all to update in this or a followup PR?

Reply:
I think it is fine